README.md

```diff
@@ -126,22 +134,24 @@ BLT bundles its external dependencies in thirdparty_builtin/.  These
 packages are covered by various permissive licenses.  A summary listing
 follows.  See the license included with each package for full details.
 
-PackageName: fruit
-PackageHomePage: https://sourceforge.net/projects/fortranxunit/
-PackageLicenseDeclared: BSD-3-Clause
+[//]: # (Note: The spaces at the end of each line below add line breaks)
```

Comment:
Thanks for adding the comment and fixing the file! 

As you noted, those are important for formatting. :)